src/index.js

```diff
@@ -1,5 +1,15 @@
 import ReactDOM from 'react-dom';
 import App from './App/App.js';
 import React from 'react';
+import { createStore } from 'redux';
+import allReducers from './Redux/reducers';
+import Provider from 'react-redux';
 
-ReactDOM.render(<App />, document.getElementById('root'));
+const store = createStore(allReducers);
+
+ReactDOM.render(
+	<Provider store={store}>
+		<App />
+	</Provider>,
+	document.getElementById('root')
```

Comment:
it will be good if you configure `Provider` under `App/App.js` as we discussed on it